.travis.yml

```diff
@@ -96,7 +96,7 @@ matrix:
         # for the image tests.
         - os: linux
           env: PYTHON_VERSION=2.7 SETUP_CMD='test --remote-data=astropy -a "--mpl"'
-               CONDA_DEPENDENCIES=$CONDA_ALL_DEPENDENCIES
+               CONDA_DEPENDENCIES="`echo $CONDA_ALL_DEPENDENCIES mkl=11.3.3`"
```

Comment:
Maybe you already tried this but I think it should be possible to just do:
```
CONDA_DEPENDENCIES="$CONDA_ALL_DEPENDENCIES mkl=11.3.3"

Reply:
I think I tried it (I've tried a few variants), and it picked up mkl as a separate env variable